Thumbnail/ConsumerThumbnail.php

```diff
@@ -23,16 +25,19 @@ class ConsumerThumbnail implements ThumbnailInterface
 
     protected $backend;
 
+    protected $dispatcher;
+
     /**
      * @param string                                              $id
      * @param ThumbnailInterface                                  $thumbnail
      * @param \Sonata\NotificationBundle\Backend\BackendInterface $backend
      */
-    public function __construct($id, ThumbnailInterface $thumbnail, BackendInterface $backend)
+    public function __construct($id, ThumbnailInterface $thumbnail, BackendInterface $backend, EventDispatcherInterface $dispatcher)
```

Comment:
Change signature by adding a required argument would be BC break.

Could you please handle it?